libs/lib_parsing/Unit_tok.ml

```diff
@@ -0,0 +1,24 @@
+(*
```

Comment:
BTW you don't need to create a separate file. You can have "inline tests" now next to the function
it covers.
See Testutil.test, and 
src/core/ATDStringWrap.ml as an example.

Reply:
That's good to know, I'll keep `Unit_tok.ml` for now, partly because I think the test code is somewhat verbose to be inlined. And it also seems that we are not using `Testutil.test` that much yet. I like the idea of inlining tests, but maybe it's better with something like `ppx_expect` ?

Reply:
I think it's orthogonal. You can have inline Testutil.test with simple asserts, and you can also have inline ppx_expect test with printing on stdout and assert the expected output. It's different ways to write tests (but both close to the code itself indeed).